data_specifications/specification.proto

```diff
@@ -378,11 +378,13 @@ message ControlFlowOverrides {
 message Specification {
   Arch arch = 1;
   OS operating_system = 2;
-  repeated Function functions = 3;
-  repeated Callsite callsites = 4;
-  repeated GlobalVariable global_variables = 5;
-  repeated Symbol symbols = 6;
-  repeated MemoryRange memory_ranges = 7;
-  ControlFlowOverrides overrides = 8;
-  map<int64, TypeSpec> type_aliases = 9;
+  string image_name = 3;
```

Comment:
just as a general thing for compatibility please in the future do not shift fields when adding to the protobuf def. add a new field with a new index